nifipy/components.py

```diff
@@ -24,7 +28,9 @@ class NifiConnection(object):
         BASE: "{url_base}/nifi-api",
         PROCESSOR: "{url_base}/nifi-api/processors/{component_id}",
         CONTROLLER_SERVICE: "{url_base}/nifi-api/controller-services/{component_id}",
-        FLOW: "{url_base}/nifi-api/flow/process-groups/{process_group}/{subpath}"
+        FLOW: "{url_base}/nifi-api/flow/process-groups/{component_id}/",
```

Comment:
@fthuerk I believe this change breaks original functionality. Could you have a look into it?

Time for unit tests... 😺

Reply:
This should be fixed now. Exept for unit tests :D